### DISCUSSION TITLE
How to prevent the sync of files deleted on the website

### DISCUSSION THREAD

**Discussion by just-a-beginner** [immich-app/immich#1788] [Q&A]
I just installed Immich and the sync is working great with iOS. Kudos to the great work!!!
But I'm wondering how I can prevent that the files I deleted on the website are not resynced again the next time I use the iOS app. 

If this is not possible, what would be the best approach to manage the photos/videos on the desktop as it's not very comfortable directly on the mobile device. So far I used the Image Capture app on macOS but it's not working very well. The Immich web-interface is way way better. :) 

Please bear with me if this was already answered... tried to find the information in the reported Issues and here under Discussions.

**Comment by MozPri** [comment]
Is there any progress on this?
I would really like to delete images from my device via the website.

I am one of those people who takes a lot of very similar pictures, then I go through them on the big screen at home and keep only the best ones. But Immich keeps re-uploading them...
